controller/handler/trial.js

```diff
@@ -121,6 +122,11 @@ function trialView (request, reply) {
                     patient.totalMissed = 0;
                 }
 
+                patient.dateStarted = moment(patient.dateStarted, sqlDateFormat)
```

Comment:
Instead of these lines it should be -

patient.dateStarted = moment(patient.dateStarted).format('YYYY-MM-DD')
patient.dateCompleted = moment(patient.dateCompleted).format('YYYY-MM-DD')

You can refer to the dashboard.js controller file as to how IRB start & end dates are displayed there for a trial.